packages/aws-rfdk/lib/deadline/lib/worker-configuration.ts

```diff
@@ -29,6 +29,45 @@ import {
   Version,
 } from './version';
 
+/**
+ * Provider for adding user data scripts
+ * Methods of this interface will be invoked in WorkerInstanceConfiguration
+ * on different stages of worker configuration
+ */
+export interface IInstanceUserDataProvider {
+  /** Method that is invoked before configuration Cloud Watch Agent. */
```

Comment:
These are still not in correct TSDoc syntax. Also a minor grammatical fix. Should be:

```ts
 /**
  * Method that is invoked before configuring the Cloud Watch Agent.
  */
```